tests/integration/test_simplified_workflows.py

```
@@ -0,0 +1,556 @@
+        # PyYAML might still be needed for other purposes, but not for chunking
+        if 'pyyaml' in workflow_content.lower():
+            assert 'context chunker' not in workflow_content.lower()
+    
+    def test_pr_agent_has_parse_comments_step(self, pr_agent_workflow):
+        """Verify the parse-comments step exists."""
+        jobs = pr_agent_workflow.get('jobs', {})
+        assert len(jobs) > 0
+        
+        # Find a job with parse-comments step
+        found_parse_step = False
+        for job_name, job_config in jobs.items():
```

Ruff (0.14.6)

Loop control variable `job_name` not used within loop body

Rename unused `job_name` to `_job_name`

(B007)